### PR TITLE
Dev rachel

### PR DESCRIPTION
**Pull request by rnackman**

### CODE REVIEW

File: src/components/CollectionFilters/LineFilter.jsx [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    const filters = this.props.filters;
+    for (let i = 0; i < filters.length; i++) {
+      if (filters[i].slug === this.props.filter.slug) {
+        return i;
+      }
+    }
+    return -1;
+  }
+
+  handleClick(event) {
```
Comment (erchaves) [inline src/components/CollectionFilters/LineFilter.jsx]: It'd be cool to make a parent class at some point to hold some of this shared logic. And then each one of these could extend the parent.

File: src/components/CollectionFilters/collectionFilters.scss [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+  -webkit-appearance: none;
+  width: 70%;
+  border: 0;
+  margin: 0 40px;
+
+  &:focus {
+    outline: none;
+  }
+
+  &::-webkit-slider-runnable-track {
```
Comment (erchaves) [inline src/components/CollectionFilters/collectionFilters.scss]: Is this collection of browser-specific css styles going to work well enough cross browser?

File: src/components/CollectionFilters/LineFilter.jsx [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+  }
+
+  filterIsApplied() {
+    const filters = this.props.filters;
+    for (let i = 0; i < filters.length; i++) {
+      if (filters[i].slug === this.props.filter.slug) {
+        return i;
+      }
+    }
+    return -1;
```
Comment (erchaves) [inline src/components/CollectionFilters/LineFilter.jsx]: Not a huge deal, but let's aim for consistency and add line breaks after blocks. This is a good guide. https://github.com/airbnb/javascript#whitespace--after-blocks

File: src/components/CollectionFilters/mobileFilters.scss [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+        height: auto;
+        min-height: 50px;
+        padding: 10px 0;
+        .btn-icon {
+          display: block;
+          width: 100%;
+          margin-bottom: 10px;
+        }
+      }
+      &:first-of-type {
```
Comment (erchaves) [inline src/components/CollectionFilters/mobileFilters.scss]: Can we try to add line breaks to separate blocks? I tend to follow this style guide
https://github.com/airbnb/css#formatting

File: src/components/CollectionFilters/LineFilter.jsx (hunk 1 above)
Comment (rnackman) [reply src/components/CollectionFilters/LineFilter.jsx]: Agree. For now, waiting to see what about these filters is unique once I have them appropriately querying ES. My sense is that they're actually the same thing, but holding off on combining them until I know that.

File: src/components/CollectionFilters/collectionFilters.scss (hunk 2 above)
Comment (rnackman) [reply src/components/CollectionFilters/collectionFilters.scss]: This is definitely a work in progress. Agree that this needs to be examined more carefully.

File: src/components/CollectionFilters/mobileFilters.scss (hunk 4 above)
Comment (rnackman) [reply src/components/CollectionFilters/mobileFilters.scss]: Sounds good!

File: src/components/CollectionFilters/LineFilter.jsx (hunk 3 above)
Comment (rnackman) [reply src/components/CollectionFilters/LineFilter.jsx]: 👍  I'll go through and make style changes where needed.

File: package.json [hunk 5]
```diff
@@ -28,6 +28,7 @@
     "react-helmet": "^5.1.3",
     "react-masonry-component": "^5.0.7",
     "react-redux": "^5.0.5",
+    "react-responsive": "^1.3.4",
```
[the remark this answers is not in the document]
Comment (rnackman) [reply package.json]: This manages situations (like the mobile filter UI) in which the DOM needs to be structured differently. We could do it with classes, but the components feel more legible to me this way.

File: src/components/CollectionFilters/ColorFilter.jsx [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+import * as FilterSetsActions from '../../actions/filterSets';
+
+class ColorFilter extends Component {
+  constructor(props) {
+    super(props);
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  filterIsApplied() {
+    const filters = this.props.filters;
```
[the remark this answers is not in the document]
Comment (rnackman) [reply src/components/CollectionFilters/ColorFilter.jsx]: Cool, thank you. It looks like there's no support for .find() in IE, though.

**Comment by erchaves** [comment]
I'm going to go ahead and merge this in! I'll update my code with it and send you a PR soon.

**Pull request merged**